pmm/pmm2-release.groovy

```diff
@@ -477,6 +477,9 @@ ENDSSH
             }
         }
         stage('Publish Docker image') {
+            agent {
+                label 'virtualbox'
```

Comment:
We don't have the agent called "viartualbox", so it won't work

Reply:
fixed